src/OSPSuite.Core/Serialization/Chart/CurveChartXmlSerializer.cs

```diff
@@ -18,6 +19,14 @@ public override void PerformMapping()
          MapEnumerable(x => x.Axes, x => x.AddAxis);
          MapEnumerable(x => x.Curves, x => x.AddCurveIfConsistent);
       }
+
+      protected override void TypedDeserialize(TCurveChart curveChart, XElement outputToDeserialize, SerializationContext context)
+      {
+         base.TypedDeserialize(curveChart, outputToDeserialize, context);
+
+         //once deserialize, we need to make sure that the curves are aware of the axis dimensions
+         curveChart.SynchronizeDataDisplayUnit();
```

Comment:
@Yuri05 This was a fairly simple stuff.  Once we have deserialize the whole chart, we need to make sure we synchronize display unit with the axis. The data is created on the fly and the value is not set properly (only on reload)